frontend/scss/components/molecules/pixi-recommendations-item.scss

```diff
@@ -84,7 +84,7 @@
   }
 
   &-body + &-tags {
-    border-top: solid 1px color('boulder');
+    border-top: solid 1px safeColor('light-gray');
```

Comment:
This is this line right? 
![image](https://user-images.githubusercontent.com/10456171/93928275-0ed9fa00-fce8-11ea-9687-2ebfd10567fb.png)

We can also restore it to 2px if preferred